jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ListTests.java

```diff
@@ -0,0 +1,52 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.gridListSection;
+import static io.github.com.pages.AngularPage.listSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.hamcrest.core.StringContains.containsString;
+
```

Comment:
There are many unused import statements in the class.
Please format file (delete unused import statements).